README

```diff
@@ -0,0 +1,26 @@
+# Sisyphus-style checksums
+
+This pipeline takes a runfolder and an include file and generates a checksum
+file compatible with [Sisyphus](https://github.com/Molmed/sisyphus).
+
+## Usage
+
+```nextflow
+nextflow run main.nf --input_folder <path_to_runfolder> --include_file <path_to_include_file> [--ignore_cache]
+```
+
+### Running on Uppmax
+When running on Uppmax, use `-profile uppmax` and specify project with `--project <uppmax project>`
+
+## Input
+- `--input_folder <path_to_runfolder`: path to the runfolder for which the
+checksums need to be generated
+- `--include_file <path_to_include_file`: determines which files should appear
+in the checksums (same format as `rsync`)
+- `--ignore-cache` whether or not to use previously computed checksums. When
+this flag is set, the pipeline will look for previously computed checksums in
```

Comment:
Isn't it the opposite, when this flag is used, previously computed checksums will be overwritten?

Reply:
Good catch! I just fixed it :)